src/components/course-enrollments/data/tests/service.test.js

```diff
@@ -1,25 +1,28 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
 import {
   fetchProgramCourseEnrollments,
   fetchEnterpriseCourseEnrollments,
 } from '../service';
-import apiClient from '../../../../apiClient';
 
-apiClient.get = jest.fn();
+jest.mock('@edx/frontend-platform/auth');
+const axiosMock = new MockAdapter(axios);
+getAuthenticatedHttpClient.mockReturnValue(axios);
+axiosMock.onAny().reply(200);
+axios.get = jest.fn();
 
 describe('course enrollments service', () => {
-  beforeEach(() => {
-    jest.resetAllMocks();
-  });
-
   it('fetches program enrollments', () => {
     const url = 'http://localhost:18000/api/program_enrollments/v1/programs/test-program-id/overview/';
     fetchProgramCourseEnrollments('test-program-id');
-    expect(apiClient.get).toBeCalledWith(url);
+    expect(axios.get).toBeCalledWith(url);
```

Comment:
This looks fine (and clearly not new in this PR) but it's worth noting that axios has another method `request` that could be called instead of the aliased methods `get`, `post`, etc. If you're using axios mock adapter in the future it might be good to use the history object on the axios mock to do assertions (https://www.npmjs.com/package/axios-mock-adapter#history).